tests/client/conftest.py

```diff
@@ -428,17 +428,6 @@ def feedback_dataset_records() -> List[FeedbackRecord]:
         ),
         FeedbackRecord(
             fields={"text": "This is a negative example", "label": "negative"},
-            responses=[
-                {
-                    "values": {
-                        "question-1": {"value": "This is a response to question 1"},
-                        "question-2": {"value": 1},
-                        "question-3": {"value": "a"},
-                        "question-4": {"value": ["a", "b"]},
-                    },
-                    "status": "submitted",
-                }
-            ],
```

Comment:
I think we should include a specific test case to check the bugfix. Not sure this change is covering it.

Reply:
This covers it, as well as ensuring robustness since the responses are optional, so we're mocking a `FeedbackRecord` with and without responses 👍🏻